docs/lake/design.md

```diff
@@ -91,8 +92,8 @@ design patterns of `git`.  In this approach,
 * a _commit_ operation is like a `git commit`,
 * and a pool _snapshot_ is like a `git checkout`.
 
-A core theme of the Zed lake design is _ergonomics_.  Given the git metaphor,
-our goal here is that the Zed lake tooling be as easy and familiar as git is
+A core theme of the Zed lake design is _ergonomics_.  Given the Git metaphor,
+our goal here is that the Zed lake tooling be as easy and familiar as Git is
```

Comment:
I've seen @nwt propose corrections to capitalize "Git" when speaking of the tool, which I support, so I'm continuing in that spirit. I also support `git` being in lowercase & fixed width when we're referencing the command that's invoked at the shell, and it seems the doc is already quite consistent on that one.